app/js/actions/connection.js

```diff
@@ -1,10 +1,7 @@
-import fetch from 'isomorphic-fetch';
-import es6Promise from 'es6-promise';
 import CryptoJS from 'crypto-js';
 
 import actions from './index';
 
-es6Promise.polyfill();
```

Comment:
yup! they also broke everything in the electron environment, which is how I noticed it.